packages/api/docker/docker-compose.yml

```diff
@@ -1,57 +1,126 @@
-version: '3.6'
 services:
-  rest:
+  post-rest:
     image: postgrest/postgrest:v9.0.0
     depends_on:
-      - db
-    restart: always
-    ports:
-      - 3000:3000/tcp
+      - db-init
     environment:
-      PGRST_DB_URI: postgres://postgres:postgres@db:5432/postgres
+      PGRST_DB_URI: $DATABASE_CONNECTION
       PGRST_DB_SCHEMAS: public,cargo
       PGRST_DB_ANON_ROLE: postgres
       PGRST_JWT_SECRET: super-secret-jwt-token-with-at-least-32-characters-long
+  db-init:
+    build:
+      dockerfile: './docker/Dockerfile'
+      context: '../'
+    entrypoint: 'psql $DATABASE_CONNECTION -v "ON_ERROR_STOP=1" -f ./db/config.sql -f ./db/tables.sql -f ./db/cargo.testing.sql -f ./db/functions.sql'
```

Comment:
I think we should have a init.sql script here. If we add a new sql file that needs to run, it will be difficult for the developer to understand the multiple places where it should be added. So, having a init.sql that would execute the other files would be rad. Not super sure how to do it, but found [this](https://stackoverflow.com/a/40667907)

Reply:
That's a good idea, especially if we start having fancier migrations - we can just hook them into the init script.